packages/ui-react/src/pages/ScreenRouting/RecommendationsScreenRouter.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { useParams } from 'react-router';
+import { useTranslation } from 'react-i18next';
+import type { Playlist } from '@jwp/ott-common/types/playlist';
+import { LIST_TYPE } from '@jwp/ott-common/src/constants';
+import { ScreenMap } from '@jwp/ott-common/src/utils/ScreenMap';
+import usePlaylist from '@jwp/ott-hooks-react/src/usePlaylist';
+
+import Loading from '../Loading/Loading';
+import ErrorPage from '../../components/ErrorPage/ErrorPage';
+import type { ScreenComponent } from '../../../types/screens';
+
+import PlaylistGrid from './sharedScreens/PlaylistGrid/PlaylistGrid';
+
+export const playlistScreenMap = new ScreenMap<Playlist, ScreenComponent<Playlist>>();
+
+playlistScreenMap.registerDefault(PlaylistGrid);
+
+const RecommendationsScreenRouter = () => {
```

Comment:
Do we need a separate router or could we re-use `<PlaylistScreenRouter />` and pass the list type along from a prop?

Reply:
So for menu items you think we could pass 'type' as a query param? We could reuse it. I was just thinking that it could be cleaner when we use a separate router. We can also tweak translations and avoid using LiveChannels screen.